### ISSUE TITLE
Fixes to model train/test and consistency

### ISSUE THREAD

**Issue by terryf82** [Data4Democracy/boston-crash-modeling#125]

**Comment by terryf82** [comment]
@bpben I'm a bit unsure of how we start on this one - maybe we should first define what we think a minimum worthwhile probability score is, and then we can address how we deal with the situation where we can't generate that?

**Comment by bpben** [comment]
Yeah, I think ROC <=.5 is a hard limit, either equal to or worse than chance.  Quick fix would be to just stop the prediction process and output error if it doesn't pass that assertion.
